### ISSUE TITLE
Load-leo.html needs work

### ISSUE THREAD

**Issue by maphew** [leo-editor/leo-editor#1551]
The [load-leo.html](http://leoeditor.com/load-leo.html) page needs some upgrading.

 - [x] fix php deprecated func error, _" Deprecated: Function split() is deprecated in /home3/speedhos/public_html/leoeditor.com/do-leo.php on line 334"_ See [this page](https://stackoverflow.com/questions/2813241/php-split-alternative) for alternatives.
  The new code uses explode instead of split.

 - [x] add a default "upload from internet page", such as leoDocs.leo.
`<input style="width:600px;" id="get_name" value="https://github.com/leo-editor/leo-editor/raw/master/leo/doc/LeoDocs.leo">`

**Comment by edreamleo** [comment]
On Sun, Mar 29, 2020 at 11:15 PM matt wilkie <notifications@github.com>
wrote:

> The load-leo.html <http://leoeditor.com/load-leo.html> page needs some
> upgrading.
>
> -[ ] fix php deprecated func error, *" Deprecated: Function split() is
> deprecated in /home3/speedhos/public_html/leoeditor.com/do-leo.php
> <http://leoeditor.com/do-leo.php> on line 334"*
>
> -[ ] add a default "upload from internet page", such as leoDocs.leo.
> <input style="width:600px;" id="get_name" value="
> https://github.com/leo-editor/leo-editor/raw/master/leo/doc/LeoDocs.leo">
>
Thanks for this. You're going to do this, right?

Why did you label it documentation?

Edward

**Comment by maphew** [comment]
yes I intend to fix it. Doc label because it was in looking at docs to fix (Readme specifically) that I discovered it. The nature of the job mutated while I was creating the issue. ;-)

**Comment by edreamleo** [comment]
Completed at 2035848 in devel and uploaded to leo-editor.com.

**Comment by maphew** [comment]
[Upload from internet] button yields "File is too large. Error uploading file" for LeoDocs.leo and any other url I try. 

The [upload from hard drive] button does work, though not using my full browser window, and visually wonky. 

I've adjusted the stylesheet some which I'll commit in shortly. Rather than use full browser window, it proved more complicated than I have time for, I opted to keep the fixed 700px width and darken the non-active page background.
